src/QueryDescription/test.js

```diff
@@ -40,48 +37,13 @@ describe('QueryDescription', () => {
     ])
     expect(query).toEqual({
       where: [
-        {
-          type: 'where',
-          left: 'col1',
-          comparison: {
-            operator: 'eq',
-            right: { value: 'val1' },
-          },
-        },
-        {
-          type: 'where',
-          left: 'col2',
-          comparison: {
-            operator: 'eq',
-            right: { value: 2 },
-          },
-        },
-        {
-          type: 'where',
-          left: 'col3',
-          comparison: {
-            operator: 'eq',
-            right: { value: true },
-          },
-        },
-        {
-          type: 'where',
-          left: 'col4',
-          comparison: {
-            operator: 'eq',
-            right: { value: false },
-          },
-        },
-        {
-          type: 'where',
-          left: 'col5',
-          comparison: {
-            operator: 'eq',
-            right: { value: null },
-          },
-        },
+        { type: 'where', left: 'col1', comparison: { operator: 'eq', right: { value: 'val1' } } },
+        { type: 'where', left: 'col2', comparison: { operator: 'eq', right: { value: 2 } } },
+        { type: 'where', left: 'col3', comparison: { operator: 'eq', right: { value: true } } },
+        { type: 'where', left: 'col4', comparison: { operator: 'eq', right: { value: false } } },
+        { type: 'where', left: 'col5', comparison: { operator: 'eq', right: { value: null } } },
```

Comment:
reformatting, nothing to see

Reply:
nothing to see in this file until a comment that says there's stuff to see :D

Reply:
Thanks, that was really helpful :)